src/wallet/rpcwallet.cpp

```diff
@@ -3129,7 +3129,6 @@ UniValue fundrawtransaction(const JSONRPCRequest& request)
                 {"change_type", UniValueType(UniValue::VSTR)},
                 {"includeWatching", UniValueType(UniValue::VBOOL)},
                 {"lockUnspents", UniValueType(UniValue::VBOOL)},
-                {"reserveChangeKey", UniValueType(UniValue::VBOOL)}, // DEPRECATED (and ignored), should be removed in 0.16 or so.
```

Comment:
Should we allow it behind `IsDeprecatedRPCEnabled()`?

Reply:
I guess this move is from deprecated to removed. So IMO no need to hide it behind `IsDeprecatedRPCEnabled()`.